layers/+tools/eaf/packages.el

```diff
@@ -48,8 +48,32 @@
     :defer t
     :init
     (progn
+      (with-eval-after-load 'eaf
+                    (let ((eaf-apps (list 'eaf-jupyter
+                                          'eaf-browser
+                                          'eaf-airshare
+                                          'eaf-file-browser
+                                          'eaf-file-manager
+                                          'eaf-file-sender
+                                          'eaf-music-player
+                                          'eaf-system-monitor
+                                          'eaf-mindmap
+                                          'eaf-org-previewer
+                                          'eaf-terminal
+                                          'eaf-netease-cloud-music
+                                          'eaf-video-player
+                                          'eaf-js-video-player
```

Comment:
The latest EAF has dropped the support to `js-video-player`, feel free to remove it.